.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
+dist: trusty
+
+go:
+  - 1.8.1
+
+before_install:
+  - echo $PATH
+  - export PATH=`echo $PATH | sed -e 's/:\/usr\/local\/clang-3.5.0\/bin//'`
+  - wget http://security.ubuntu.com/ubuntu/pool/main/g/gcc-5/libstdc++6_5.4.0-6ubuntu1~16.04.4_amd64.deb
+  - mkdir ./libc
+  - dpkg-deb -x ./libstdc++6_5.4.0-6ubuntu1~16.04.4_amd64.deb ./libc
+  - export LD_LIBRARY_PATH=`pwd`/libc/usr/lib/x86_64-linux-gnu/
+  - wget http://releases.llvm.org/4.0.1/clang+llvm-4.0.1-x86_64-linux-gnu-debian8.tar.xz
+  - tar -xf ./clang+llvm-4.0.1-x86_64-linux-gnu-debian8.tar.xz
+  - sudo cp clang+llvm-4.0.1-x86_64-linux-gnu-debian8/bin/clang-format /usr/local/bin/
+  - which clang-format
+  - clang-format --version
+
+install: true
+
+script:
+  - go get ./...
```

Comment:
Why is this required? We now don't have any external dependencies (everything should be in vendor/). So assuming that the system gave us the source at the right revision, we don't need to do anything else.
I can imagine that this will actually break things, or misattribute failures. Consider that it is testing one commit, but HEAD has already moved. go get will update sources to HEAD, so it will test a wrong one.
Not sure what will happen if it is testing a pull request.